examples/misc/packed_bubbles.py

```diff
@@ -0,0 +1,176 @@
+"""
+Create a packed bubble / non overlapping bubble chart to represent scalar data.
```

Comment:
The section title has vanished again.

I assume this was not intentional. Please let us know if you need help with git/github workflow for updating pull requests.

Reply:
I don't get how I pull the changes I committed on github.com to my local repo. I did a pull but it seems like nothing arrived in my local repo.

Reply:
What make it even weirder for me is the fact that the title disappeared in 7e1598de46260e3e810b151bf557fe90b0418c3b without showing up as diff.

Reply:
I usually do `git fetch origin`, and then in the branch `git rebase origin/<branch-name>`.  If I've worked locally on the branch there may be conflicts that need to be resolved.